### ISSUE TITLE
NVDA gets stuck on profile page navigating to notes

### ISSUE THREAD

**Issue by Auroratide** [Auroratide/sifetti#7]
When I navigate using NVDA on the profile page to a note page, it inexplicably gets stuck on the profile page while the rest of the page visibly loads (reads "My Profile" over and over again). It's like it doesn't understand that we're on a new page now.

The broader behaviour should be that when we navigate with a screen reader from one page to another that the cursor lands at the very top, and ideally there would be a Skip Navigation link up there.
